.github/workflows/make-apk.yml

```
@@ -7,14 +7,24 @@ on:
 
 jobs:
   Build-Apk:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-20.04 # That is the latest OS version available today
```

Is here any step that will stop working on future versions ? Otherwise, this should be fine:
```suggestion
    runs-on: ubuntu-latest
```

---

That line was there before me, I only added the comment because I did tried to run ubuntu 21.10 to get a newer version of fonforge, and then I found out that v20.04 was the latest available for actions. As of today, pining the version doesn't change anything as it is also the latest. If in the future there is some breaking change, that should be fixed anyway.

---

I added it because of the patching of the sources.list, so can be removed.